src/lib/features/export-import-toggles/export-import-api-usage.test.ts

```diff
@@ -0,0 +1,182 @@
+import { setupAppWithCustomAuth } from '../../../test/e2e/helpers/test-helper';
+import dbInit, { ITestDb } from '../../../test/e2e/helpers/database-init';
+import getLogger from '../../../test/fixtures/no-logger';
+import {
+    DEFAULT_PROJECT,
+    IContextFieldStore,
+    IEnvironmentStore,
+    IEventStore,
+    IFeatureToggleStore,
+    IProjectStore,
+    ITagStore,
+} from '../../types';
+import { DEFAULT_ENV } from '../../util';
+import { ImportTogglesSchema, VariantsSchema } from '../../openapi';
+import { ApiTokenType } from '../../types/models/api-token';
+import { ApiUser } from '../../server-impl';
+
+let db: ITestDb;
+let eventStore: IEventStore;
+let environmentStore: IEnvironmentStore;
+let contextFieldStore: IContextFieldStore;
+let projectStore: IProjectStore;
+let toggleStore: IFeatureToggleStore;
+let tagStore: ITagStore;
+
+beforeAll(async () => {
+    db = await dbInit('export_import_api_admin', getLogger);
+    eventStore = db.stores.eventStore;
+    environmentStore = db.stores.environmentStore;
+    projectStore = db.stores.projectStore;
+    contextFieldStore = db.stores.contextFieldStore;
+    toggleStore = db.stores.featureToggleStore;
+    tagStore = db.stores.tagStore;
+});
+
+beforeEach(async () => {
+    await eventStore.deleteAll();
+    await toggleStore.deleteAll();
+    await projectStore.deleteAll();
+    await environmentStore.deleteAll();
+    await tagStore.deleteAll();
+
+    await contextFieldStore.deleteAll();
+});
+
+afterAll(async () => {
+    await db.destroy();
+});
+
+const defaultFeature = 'first_feature';
+
+const variants: VariantsSchema = [
```

Comment:
I'd craft a minimal input to test the scenario and pass the input validation. E.g. to test the import is rejected we don't need those variants.